lib/apiary/version.rb

```diff
@@ -1,3 +1,3 @@
 module Apiary
-  VERSION = '0.6.1'.freeze
+  VERSION = '0.6.2'.freeze
```

Comment:
If this is following semver, it should bump to `0.7.0` because it's adding functionality